openwisp_monitoring/monitoring/base/models.py

```diff
@@ -145,7 +145,7 @@ def content_type_key(self):
         except AttributeError:
             return None
 
-    def check_threshold(self, value, time=None, retention_policy=None):
+    def check_threshold(self, value, time=None, retention_policy=None, send_alert=True):
```

Comment:
named this parameter as `send_alert` instead of `notify` to avoid any confusion with a signal of the same name.